package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dash-cytoscape",
-  "version": "0.0.1",
```

Comment:
You'll need to update this one too: https://github.com/plotly/dash-cytoscape/blob/master/dash_cytoscape/package.json

Or, you can just run `build:py` which will copy `package.json` into `dash_cytoscape/package.json`

Reply:
Yes it's changed now